electrum_nmc/electrum/commands.py

```diff
@@ -1273,6 +1276,35 @@ async def add_request(self, amount, memo='', expiration=3600, force=False, walle
         wallet.save_db()
         return wallet.export_request(req)
 
+    @command('w')
+    async def add_name_request(self, identifier, amount=0, memo='', expiration=3600, force=False, wallet: Abstract_Wallet = None):
```

Comment:
Why not just add `identifier` as an extra optional parameter to `add_request`?

Reply:
The user must enter 0 if they choose not to provide an extra amount.
We can just add an instruction for this